libs/platform/auth/src/components/logout/logout.def.ts

```diff
@@ -9,7 +9,7 @@ declare global {
 
 export const authLogoutComponent = componentDef({
   name: 'oryx-auth-logout',
-  impl: () => import('./logout.component').then((m) => m.AuthLogoutComponent),
+  impl: () => import('./logout.component').then((m) => m.default),
```

Comment:
Why `default`?

Reply:
Actually I would like to propose to use default for our dynamic imports to reduce amount of code needed to be written, and to add default extraction to the loader itself, so we can simply keep `import('./logout.component')` and no `.then()` chain is required.
So this was just a preparation for that stage =)

Reply:
we should discuss it for sure

Reply:
Not a big deal to merge this PR now, but I agree we need to discuss this one. 

I wonder if there are any consequences. For example, in some cases we might have multiple exports in the component fil.

Reply:
Anyway, let's rol it back as it's not related to auth topic

Reply:
(i'm doing it so we can resolve this one and merge)